models/index.ts

```diff
@@ -14,6 +14,15 @@ export type AppVersionPageQuery = AppVersionProps & {
   selectedTab?: typeof AppVersionPageTabs[number];
 };
 
+export type AppPageQuery = {
+  appSlug: string;
+};
+
+export type AppVersionList = {
+  version: string;
+  apps: AppVersion[]
```

Comment:
These are app versions, not apps
```suggestion
  appVersions: AppVersion[]
```